lms/templates/dashboard/_dashboard_credit_information.html

```diff
@@ -0,0 +1,48 @@
+<%page args="credit_message, course" />
+
+<%!
+from django.utils.translation import ugettext as _
+from course_modes.models import CourseMode
+%>
+<%namespace name='static' file='../static_content.html'/>
+
+<div class="message message-status is-shown">
+    <p>
+        % if credit_message["status"] == "requirements_meet":
+            <span>
+            % if credit_message["urgent"]:
+                ${_("{username}, your eligibility for credit expires on {expiry}. Don't miss out!").format(
+                        username=credit_message["user_full_name"],
+                        expiry=credit_message["expiry"]
```

Comment:
You can use this helper function: https://github.com/edx/edx-platform/blob/master/common/djangoapps/util/date_utils.py#L12